ownership-chain/runtime/laos/src/configs/fee.rs

```diff
@@ -0,0 +1,201 @@
+		type MaxHolds = ();
+		type MaxFreezes = ();
+	}
+	impl pallet_authorship::Config for Test {
+		type FindAuthor = AuthorGiven;
+		type EventHandler = ();
+	}
+
+	const TEST_ID: ConsensusEngineId = [1, 2, 3, 4];
+
+	pub struct AuthorGiven;
+
+	impl frame_support::traits::FindAuthor<u64> for AuthorGiven {
+		fn find_author<'a, I>(digests: I) -> Option<u64>
+		where
+			I: 'a + IntoIterator<Item = (ConsensusEngineId, &'a [u8])>,
+		{
+			for (id, mut data) in digests {
+				if id == TEST_ID {
+					return u64::decode(&mut data).ok()
+				}
+			}
+
+			None
+		}
+	}
+
+	fn seal_header(mut header: Header, author: u64) -> Header {
+		{
+			let digest = header.digest_mut();
+			digest.logs.push(DigestItem::PreRuntime(TEST_ID, author.encode()));
+			digest.logs.push(DigestItem::Seal(TEST_ID, author.encode()));
+		}
+
+		header
+	}
+
+	fn create_header(number: u64, parent_hash: H256, state_root: H256) -> Header {
+		Header::new(number, Default::default(), state_root, parent_hash, Default::default())
+	}
+
+	fn new_test_ext() -> sp_io::TestExternalities {
+		let t = frame_system::GenesisConfig::<Test>::default().build_storage().unwrap();
+		t.into()
+	}
+
+	fn initialize_block_and_set_author(number: u64, author: AccountId) {
+		let mut header =
+			seal_header(create_header(number, Default::default(), [1; 32].into()), author);
+
+		header.digest_mut().pop(); // pop the seal off.
+		System::reset_events();
+		System::initialize(&number, &Default::default(), header.digest());
+	}
+
+	#[test]
+	fn sets_author_lazily() {
+		new_test_ext().execute_with(|| {
+			let author = 42;
+
+			initialize_block_and_set_author(1, author);
+			assert_eq!(Authorship::author(), Some(author));
+		});
+	}
+
+	#[test]
+	fn all_fee_should_go_to_block_author() {
+		new_test_ext().execute_with(|| {
+			let fee_amount = 100;
+			let author = 62;
+
+			initialize_block_and_set_author(1, author);
+
+			// initial author balance
+			let initial_author_balance = pallet_balances::Pallet::<Test>::free_balance(author);
+
+			// Mock the creation of a negative imbalance of 100 units
+			let imbalance = pallet_balances::NegativeImbalance::new(fee_amount);
+
+			// Distribute the fees
+			ToAuthor::<Test>::on_unbalanceds(vec![imbalance].into_iter());
+
+			// Assert the expected state of balances after distribution
+			let author_balance = pallet_balances::Pallet::<Test>::free_balance(author);
+
+			// Assuming all fees are distributed to the author
+			let expected_author_balance = initial_author_balance + fee_amount;
+
+			assert_eq!(
+				author_balance, expected_author_balance,
+				"Author did not receive the correct amount"
+			);
+		});
+	}
+
+	#[test]
+	fn with_no_author_fee_should_be_burned() {
+		new_test_ext().execute_with(|| {
+			let fee_amount = 100;
+
+			assert_eq!(
+				pallet_authorship::Pallet::<Test>::author(),
+				None,
+				"Author should not be set"
+			);
+
+			assert!(pallet_balances::Pallet::<Test>::deposit(
+				&66,
+				fee_amount * 2,
```

Comment:
guessing that `fee_amount * 2` serves as an arbitrary amount, or did you set this specific amount for any reason?